web/firefoxcom.js

```diff
@@ -115,7 +115,7 @@ class DownloadManager {
     );
   }
 
-  download(blob, url, filename) {
+  download(blob, url, filename, isFallback) {
```

Comment:
Please change this to the following, to clearly mark the parameter as optional:
```suggestion
  download(blob, url, filename, isFallback = false) {
```